relay-auth/src/lib.rs

```diff
@@ -687,6 +690,10 @@ impl RegisterResponse {
     pub fn token(&self) -> &str {
         self.token.as_str()
     }
+
+    pub fn version(&self) -> String {
```

Comment:
_nit_: Expose the version and have the caller stringify or do other things to it.

```suggestion
    pub fn version(&self) -> RelayVersion {
```

Reply:
ok